build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
The `galvo-dom-*` libraries have a small test suite that runs in a browser: https://github.com/standard-library/galvo-dom-clicker/blob/master/test/index.js

It would be nice to get a basic test set up that has a handful of tall elements, calls `illume` and then scrolls to different parts of the page and tests which element is marked as active with some class name. Kind of a big integration test.